pyomo/gdp/disjunct.py

```diff
@@ -44,6 +47,272 @@ class GDP_Error(PyomoException):
     """Exception raised while processing GDP Models"""
 
 
+class AutoLinkedBinaryVar(SimpleVar):
+    """A binary variable implicitly linked to its equivalent Boolean variable.
+
+    Basic oprerations like setting values and fixing/unfixing this
+    variable are also automatically applied to the assiciated binary
+    variable.
+
+    As this class is only intended to provide a deprecation path for
+    Disjunct.indicator_var, it only supports Scalar instances and does
+    not support indexing.
+    """
+
+    INTEGER_TOLERANCE = 0.001
```

Comment:
Is this the right place for such an integer tolerance? If so, should GDPOpt use this same value?

Reply:
@bernalde Good question.  I put it here mostly because it governs how these two classes interact (so, e.g., pulling in a tolerance from `contrib` didn't make sense).  That said, I picked the number arbitrarily (I think the though process was "1e-2 seems too big, and 1e-4 is probably too close to a solver's integer tolerance.  Let's go with 1e-3 and see if anyone complains.") and am open to changing it.